.github/workflows/benchmark-engine.yml

```diff
@@ -23,13 +23,21 @@ jobs:
   benchdev:
     if: ${{ github.repository == 'dagger/dagger' && (github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'benchmark')) }}
     runs-on:
-        - 'dagger-g3-v0-15-4-16c-st-benchmark'
+      - "dagger-g3-v0-15-4-16c-st-benchmark"
     timeout-minutes: 10
     steps:
       - name: Checkout repo
         uses: actions/checkout@v4
       - name: Warm up Engine
-        run: dagger core version
+        run: |
+          # Detect if a dev engine is available, if so: use that
+          # We don't rely on PATH because the GHA runner messes with that
+          if [[ -n "$_EXPERIMENTAL_DAGGER_CLI_BIN" ]]; then
+            ls -lh $(dirname $_EXPERIMENTAL_DAGGER_CLI_BIN)
+            export PATH=$(dirname "$_EXPERIMENTAL_DAGGER_CLI_BIN"):$PATH
+          fi
```

Comment:
how would there be a dev engine here? there's nothing resembling a build ahead of this step?

Reply:
the (initial?) intent here is not to have `dev-` equivalent tests, both on main and on branches the outside engine is always the released version

Reply:
from the #9477 where i'm trying to fix the notifications for the benchmark jobs...

https://github.com/dagger/dagger/actions/runs/13377712071/job/37361212417?pr=9477

reports v0.16.0-longdevhash in the warmup, but is running on a 15.4 runner, so clearly i'm confused about something here

Reply:
Ah yeah this was just a shot in the dark, I will undo it. I think we could most likely just wait this out for v0.16.0, once our CI runners are upgraded to it this should go away afaik